src/figuresClasses.ts

```diff
@@ -1,11 +1,94 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string;
+
+  color: string;
+
+  private a: number;
+
+  private b: number;
+
+  private c: number;
+
+  constructor(color: string, a: number, b: number, c: number) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('error');
+    }
+
+    if (a >= b + c || b >= a + c || c >= a + b) {
+      throw new Error('error');
```

Comment:
The error message should explain the specific issue. For instance: 'The sum of any two sides must be greater than the third side.'